packages/oui-button/_variables.less

```diff
@@ -41,8 +43,14 @@
 
 // Button - Icon Right
 @oui-button-icon-right-icon-color_disabled: @oui-button-filled-font-color_disabled;
-@oui-button-icon-right-icon-margin-left: ~"calc(@{oui-button-icon-icon-size} / 2)";
+@oui-button-icon-right-icon-margin-left: ~"calc(@{oui-button-icon-icon-size} / 3)";
```

Comment:
It's an arbitrary value to define a spacing between the label and the icon that is proportional to the icon height.